Makefile

```diff
@@ -60,7 +60,7 @@ endif
 ldflags += $(LDFLAGS)
 ldflags := $(strip $(ldflags))
 
-BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)' -trimpath
+BUILD_FLAGS := -tags "$(build_tags_comma_sep)" -ldflags '$(ldflags)' -trimpath
```

Comment:
I removed the quotes because this is comma separated. If we need quotes, we should use the `'` like in the next argument.